presto-main/src/main/java/com/facebook/presto/execution/QueryStateMachine.java

```diff
@@ -482,6 +482,7 @@ public QueryInfo getQueryInfo(Optional<StageInfo> rootStage)
                 addedSessionFunctions,
                 removedSessionFunctions,
                 Optional.ofNullable(planStatsAndCosts.get()).orElseGet(StatsAndCosts::empty),
+                session.getOptimizerInformationCollector().getInformationJson(),
```

Comment:
is there a reason to convert it to a json here? Why not send to the QueryCompletedEvent as a map, and then let whatever is logging the even figure out how they want to represent it?

Reply:
No specific reasons. I just want to have the information to be logged as Json in the end, as this enable us to add more field without schema change in destination table. Just updated the code to pass raw information, and will have the Json conversion when we do the real logging work.